index.bs

```diff
@@ -6596,6 +6626,18 @@ The following abstract operations are a grab-bag of utilities.
  1. Return ? [$StructuredDeserialize$](|serialized|, [=the current Realm=]).
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="StructuredTransferOrSerialize">StructuredTransferOrSerialize(|value|)</dfn>
```

Comment:
Maybe "StructuredTransferOrClone" would be a clearer name?

Reply:
ok